tests/test_docx.py

```diff
@@ -45,8 +45,6 @@ class ConvertDocxToHtmlTestCase(TestCase):
         # level upperRomans are converted.
         'external_image',
         'export_from_googledocs',
-        'fake_subscript',
```

Comment:
I feel like we should not be removing these test cases. We historically have at least one test case that uses a real docx to make sure our broad assumptions about docx files are valid.